microsoft-365/enterprise/cross-tenant-mailbox-migration.md

```diff
@@ -396,7 +396,11 @@ The meetings will move, however the Teams meeting URL does not update when items
 
 ### Does the Teams chat folder content migrate cross-tenant?
 
-No, the Teams chat folder content does not migrate cross-tenant. When a mailbox is migrated cross-tenant with this feature, only user visible content in the mailbox (email, contacts, calendar, tasks, and notes) is migrated.
+No, the Teams chat folder content does not migrate cross-tenant. However, once the mailbox has been migrated cross-tenant, the Teams chat folder content will be available for source tenant admins to search and export using a content search.
+
+### What content is migrated cross-tenant then?
```

Comment:
Let's remove the then and move this above the chat folder and remove the Then.

Reply:
done